src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs

```diff
@@ -84,6 +85,14 @@ public async Task<TitleViewModel> GetTitleViewModelAsync(TitleEditModel m)
 
         public async Task<OrchestratorResponse<Guid>> PostTitleEditModelAsync(TitleEditModel m, VacancyUser user)
         {
+            TrainingProvider provider = null;
+            IApprenticeshipProgramme programme = null;
+            if (IsReferredFromMaHomeFavourites(m))
+            {
+                provider = await GetProvider(m.ReferredFromMAHome_UKPRN);
+                programme = await GetProgramme(m.ReferredFromMAHome_ProgrammeId);
```

Comment:
What happen's if ether of these returns null?

Reply:
Just null would get passed in.